metadata-db-app/src/cmr/metadata_db/services/messages.clj

```diff
@@ -209,6 +209,11 @@
   (format "Humanizer could not be associated with provider [%s]. Humanizer is system level entity."
           provider-id))
 
+(defn subscriptions-only-system-level
+  [provider-id]
+  (format "Subscription could not be associated with provider [%s]. Subscription is system level entity."
```

Comment:
This is a strange error message, are we accepting a provider-id param? If this is only system level, then why would we even know about provider.  Shouldn't we throw an error farther up saying parameter not recognized or something similar?

Reply:
Look at line 209 above. this is done the same as other system level concepts.

Reply:
Weird